userspace/libsinsp/cgroup_list_counter.h

```diff
@@ -68,7 +68,7 @@ class cgroup_list_counter
 		}
 		catch (const std::exception& ex)
 		{
-			g_logger.format(log_level,
+			g_logger.format(sinsp_logger::SEV_ERROR,
```

Comment:
Will we have any context about what `list` has an `invalid format`?

Reply:
Like to know which cgroup it came from?

We'd need DEBUG logs. Think that is good enough or should I pass in "context"? I think this is pretty unlikely to happen.

Reply:
should be OK, I'd just not want to debug "Well, it failed" later :)